packages/gatsby/src/schema/__tests__/queries.js

```diff
@@ -142,8 +142,8 @@ describe(`Query schema`, () => {
     })
 
     store.dispatch({ type: `DELETE_CACHE` })
-    nodes.forEach(node =>
-      actions.createNode(node, { name: `test` })(store.dispatch)
+    getTestNodes().forEach(node =>
+      actions.createNode({ ...node }, { name: `test` })(store.dispatch)
```

Comment:
just making sure we don't strongly hold same object in memory, making it impossible to test Node.js possibly releasing nodes from memory